src/EditorFeatures/Core/Extensibility/Commands/PredefinedCommandHandlerNames.cs

```diff
@@ -171,5 +171,10 @@ internal static class PredefinedCommandHandlerNames
         /// Command handler name for Edit and Continue file save handler.
         /// </summary>
         public const string EditAndContinueFileSave = "Edit and Continue Save File Handler";
+
+        /// <summary>
+        /// Command handler name for the LSP client Signature Help command handler.
+        /// </summary>
+        public const string LSPSignatureHelp = "LSP SignatureHelpCommandHandler";
```

Comment:
This class was where we were defining names for our own command handlers; can we make a different class for external ones? Or just inline the string?